test/DynamoCoreWpfTests/PreferencesViewModelTests.cs

```diff
@@ -136,5 +139,30 @@ public void TestGetTransformedHostUnits()
             Assert.AreEqual(ViewModel.PreferencesViewModel.GetTransformedHostUnits(Dynamo.Configuration.Configurations.Units.Miles),
                 Dynamo.Configuration.Configurations.Units.Miles);
         }
+
+        [Test]
+        public void PathManagerWithDifferentHostTest()
+        {
+            PathManager singletonPathManager = PathManager.Instance;
```

Comment:
will this now affect all the other tests in this fixture?

Reply:
> will this now affect all the other tests in this fixture?

No because all the tests pass the DefaultStartConfiguration StartInTestMode true in their base classes, if the DynamoModel receives that configuration it will work with a non singleton PathManager. This is the only test that works with the Singleton PathManager.